### PR TITLE
Bbkk 9

### PR DESCRIPTION
**Pull request by DamianSz2000**
/show_beton implementation

### CODE REVIEW

File: betoniarzClicker/BetonClicker.py [hunk 1]
```diff
@@ -18,4 +18,16 @@ async def mine(self, interaction: discord.Interaction):
         self.betonClickerDB.add_beton(interaction.user.id, mined_beton)
         await interaction.response.send_message(f"<@{interaction.user.id}> kopie b b b b beton!")
         await interaction.followup.send(f"<@{interaction.user.id}> wykopał {mined_beton} betonu!")
+    async def show_beton(self, interaction: discord.Interaction):
+        if self.betonClickerDB.record(f"SELECT * FROM users WHERE UserID = ?", interaction.user.id):
+            data = self.betonClickerDB.record(f"SELECT * FROM users WHERE UserID = ?", interaction.user.id)
+        else:
+            self.betonClickerDB.execute(f"INSERT INTO users VALUES (?, ?, ?, ?)", interaction.user.id, 0, 0, 0)
+            data = self.betonClickerDB.record(f"SELECT * FROM users WHERE UserID = ?", interaction.user.id)
```
Comment (Montysz) [inline betoniarzClicker/BetonClicker.py:26]: What if databse will change a fields?

File: betoniarzClicker/BetonClicker.py [hunk 2]
```diff
@@ -18,4 +18,16 @@ async def mine(self, interaction: discord.Interaction):
         self.betonClickerDB.add_beton(interaction.user.id, mined_beton)
         await interaction.response.send_message(f"<@{interaction.user.id}> kopie b b b b beton!")
         await interaction.followup.send(f"<@{interaction.user.id}> wykopał {mined_beton} betonu!")
+    async def show_beton(self, interaction: discord.Interaction):
+        if self.betonClickerDB.record(f"SELECT * FROM users WHERE UserID = ?", interaction.user.id):
+            data = self.betonClickerDB.record(f"SELECT * FROM users WHERE UserID = ?", interaction.user.id)
```
Comment (Montysz) [inline betoniarzClicker/BetonClicker.py:23]: Double same query

File: betoniarzClicker/BetonClicker.py (hunk 1 above)
Comment (DamianSz2000) [reply betoniarzClicker/BetonClicker.py]: [#92](https://github.com/PawelPrusisz/Betoniarz-bot/pull/6/commits/92d0f70ba785bffb2d6b9071357aa03be329c530)

**Review by PawelPrusisz — APPROVED** [review]
+2

**Pull request merged**